packages/modul-components/src/components/spinner/spinner.ts

```diff
@@ -65,35 +63,31 @@ export class MSpinner extends ModulVue {
     private portalTargetEl: HTMLElement | undefined = {} as HTMLElement; // initialized to be responsive
 
     private initialized: boolean = false; // seems to be necessary since $refs are not responsive
-    private visible: boolean = false;
     private stackId: string;
 
     protected created(): void {
         this.portalTargetEl = undefined;
     }
 
     protected beforeDestroy(): void {
-        if (this.processing) {
-            let el: HTMLElement = document.getElementById(this.spinnerId) as HTMLElement;
-            if (el) {
-                document.body.removeChild(el);
-            }
-        }
+        this.removeBackdrop();
```

Comment:
on devrait peut etre laisser le if (this.processing) { non ?

Reply:
J'ai ajouté du code dans la méthode removeBackdrop() pour être sûr que tous les éléments liés au portal sont supprimés au beforeDestroy()